includes/class-wc-gateway-stripe-addons.php

```diff
@@ -149,6 +149,8 @@ public function process_subscription_payment( $order = '', $amount = 0 ) {
 					'site_url'       => esc_url( get_site_url() ),
 				);
 				$response          = WC_Stripe_API::request( $request );
+			} elseif ( ! empty( $response->errors ) ) {
```

Comment:
we could simplify this with just `else` as when `WP_Error` is instantiated it has, at least, one error stored.